cloudbuild.yaml

```diff
@@ -0,0 +1,68 @@
+timeout: 3600s # 1 hour
+logsBucket: 'gs://oppia-cloudbuild-logs'
+steps:
+  - id: build-oppia-base
+    name: gcr.io/kaniko-project/executor:latest
+    wait_for: ["-"]
+    args: 
+    - --dockerfile=Dockerfile.base
+    - --destination=gcr.io/oppia-jenkins-test-server/oppia-base:$SHORT_SHA
```

Comment:
Similarly, who is in charge of this container? Ditto below

Reply:
This container is hosted in our Google Cloud project, so we're in charge of it. The project we were using to test this out was called `oppia-jenkins-test-server` even though we're not using Jenkins